ietf/sync/tests.py

```diff
@@ -30,19 +30,20 @@
 class IANASyncTests(TestCase):
     def test_protocol_page_sync(self):
         draft = WgDraftFactory()
-        DocAlias.objects.create(name="rfc1234").docs.add(draft)
-        DocEvent.objects.create(doc=draft, rev=draft.rev, type="published_rfc", by=Person.objects.get(name="(System)"))
+        rfc = RfcFactory(rfc_number=1234)
+        draft.relateddocument_set.create(relationship_id="became_rfc", target = rfc)
```

Comment:
Thought for the future - we should probably add a `draft` parameter to the `RfcFactory` classes that handles the linkages so that we don't wind up with a bunch of subtly different ways of accomplishing this.

Reply:
Definitely. A bigger cleanup task though.